chapter-02/3_1_phrasebook/main_internal_test.go

```diff
@@ -10,52 +10,52 @@ func ExampleMain() {
 
 func TestGreet(t *testing.T) {
 	type testCase struct {
-		lang             language
-		expectedGreeting string
+		lang language
+		want string
 	}
 
 	var tests = map[string]testCase{
 		"English": {
-			lang:             "en",
-			expectedGreeting: "Hello world",
+			lang: "en",
+			want: "Hello world",
 		},
 		"French": {
-			lang:             "fr",
-			expectedGreeting: "Bonjour le monde",
+			lang: "fr",
+			want: "Bonjour le monde",
 		},
 		"Akkadian, not supported": {
-			lang:             "akk",
-			expectedGreeting: `unsupported language: "akk"`,
+			lang: "akk",
+			want: `unsupported language: "akk"`,
 		},
 		"Greek": {
-			lang:             "el",
-			expectedGreeting: "Χαίρετε Κόσμε",
+			lang: "el",
+			want: "Χαίρετε Κόσμε",
```

Comment:
I wonder if this is correct. Ancient greek wouldn't use `Χαίρετε` for a single `Κόσμε`. 
I think `Χαῖρε, Κόσμε` would be more correct.

Reply:
Would you like to check that with native speakers?

Reply:
![image](https://user-images.githubusercontent.com/11455210/225312214-9a05f576-3e11-4194-915a-ada9fc372255.png)